source/ucumXmlDocument.js

```diff
@@ -291,7 +290,7 @@ export class UcumXmlDocument {
           attrs['baseFactor_'] = parseFloat(valNode.attr.value) ;
         }
         else {
-          attrs['baseFactor_'] = valNode.val;
+          attrs['baseFactor_'] = parseFloat(valNode.val);
```

Comment:
Isn't this already a number?

Reply:
No, `typeof valNode.val === 'string'`. This is the text value (between the angle brackets) of the XmlElement.